server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsExciStartBrowseStatement.java

```diff
@@ -14,36 +14,83 @@
  */
 package org.eclipse.lsp.cobol.usecases;
 
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import org.eclipse.lsp.cobol.test.engine.UseCaseEngine;
+import org.eclipse.lsp.cobol.common.error.ErrorSource;
+import org.eclipse.lsp.cobol.usecases.common.CICSTestUtils;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4j.Range;
 import org.junit.jupiter.api.Test;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * Tests CICS STARTBROWSE CONTAINER (EXCI) statement. Ref:
  * https://www.ibm.com/docs/en/cics-ts/6.1?topic=interface-exec-cics-link-command-exci
  */
 public class TestCicsExciStartBrowseStatement {
-  public static final String TEXT =
-      "       IDENTIFICATION DIVISION.\n"
-          + "       PROGRAM-ID. EXCISTMTTEST.\n"
+
+  // Main Building Blocks
+  private static final String BASE_TEXT =
```

Comment:
Remove the redundant template as you are already using `CICSTestUtils.noErrorTest` or `CICSTestUtils.errorTest`

Reply:
Done in latest commit.